### PR TITLE
made truckmonth.py objectively better

### PR DESCRIPTION
**Pull request by Narfee**
much more dynamic, easier to scale.

### CODE REVIEW

File: neo/utils/truck_month.py [hunk 1]
```diff
@@ -11,7 +11,8 @@ def get_next_year(current_time: datetime) -> int:
 
 
 def is_it_before_truck_month_of_the_current_year(current_time: datetime) -> bool:
-    if current_time.month < 2:
+    the_number_two = "2"
+    if current_time.month < int("the_number_two"):
```
Comment (Narfee) [inline neo/utils/truck_month.py]: intentionally broken

Comment (Narfee) [reply neo/utils/truck_month.py]: yes, very intentionally

**Review by Narfee — COMMENTED** [review]
incredible work narfee

**Comment by nickofolas** [comment]
Amazing, thank you for your hard work and dedication to improving the code

**Pull request merged**